cmd-replay.c

```diff
@@ -239,6 +239,13 @@ static void setup_field(struct opts *opts)
 		s++;
 	}
 
+	if (opts->range.start > 0) {
+		if (opts->range.start_elapsed)
+			add_field(field_table[5]);
+		else
+			add_field(field_table[3]);
+	}
```

Comment:
It seems like you missed my comment on this: you need to check opts->range.stop also.  And I think you need to change default value of opts->fields (to `NULL`?) so that you can determine whether the field string is given by user or not.  If user gives field option, then it should ignore this logic and show the user-given fields only.